src/com/facebook/buck/android/UnzipAar.java

```diff
@@ -203,7 +213,7 @@ public BuildOutput initializeFromDisk(OnDiskBuildInfo onDiskBuildInfo) throws IO
   @Override
   @Nullable
   public Path getPathToOutput() {
-    return null;
+    return copiedAar;
```

Comment:
Yeah, it does feel a little janky.  I guess we could graph enhance another step in there to copy the input first and then pass that to the prebuilt jar and aar?